dbclient/dbclient.py

```diff
@@ -33,8 +36,11 @@ class dbclient:
     """
     Rest API Wrapper for Databricks APIs
     """
-    # set of http error codes to throw an exception if hit. Handles client and auth errors
-    http_error_codes = [401, 429]
+    # set of http error codes to retry
+    http_retry_codes = [429, 503]
```

Comment:
on `http_retry_codes` and `http_error_codes`, can we leave comments on what each of the code we are handling? (for more explicit readability)

Reply:
hmm, if we are retrying on 503, shouldn't we also retry on 504?

Reply:
fixed https://github.com/databrickslabs/migrate/pull/142/commits/6f05b385f2f470223f61c712d5651cdb59cea5dd